core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -115,7 +115,11 @@
   //
   // Master related properties
   //
-  MASTER_ADDRESS(Name.MASTER_ADDRESS, null),
+  /**
```

Comment:
remove this javadoc and instead do:

`@Deprecated MASTER_ADDRESS(Name.MASTER_ADDRESS, null), // use MASTER_HOSTNAME instead`